lib/hyrax/transactions/steps/set_default_admin_set.rb

```diff
@@ -3,21 +3,22 @@ module Hyrax
   module Transactions
     module Steps
       ##
-      # A `dry-transaction` step that sets the `AdminSet` for an input work to
-      # the default admin set, if none is already set. Creates the default
-      # admin set if it doesn't already exist.
+      # A step that sets the `AdminSet` for an input work to the default admin
+      # set if none is already set. Creates the default admin set if it doesn't
+      # already exist.
+      #
+      # @since 2.4.0
```

Comment:
As this Step already exists, I'm okay with it. I do like the more conditional naming `SetDefaultAdminSetUnlessAlreadySet` class, but can see maintaining the previous implemented name.